test/collect_eigen.jl

```diff
@@ -13,7 +13,7 @@ using LinearAlgebra
         R = triu(rand(T, n, n))
 
         # Compute exact eigenvectors according to LAPACK
-        λs, xs = eigen(R)
+        λs, xs = eigen(R, sortby=nothing)
```

Comment:
This keyword is only accepted as of 1.2.0-DEV.275, so I think you might need a `VERSION` conditional

Reply:
Yes. Just realized. Should be fixed now.